catkin_tools/verbs/catkin_build/cli.py

```diff
@@ -79,9 +79,11 @@ def prepare_arguments(parser):
         help='Build a given package and those which depend on it, skipping any before it.')
     add('--start-with-this', action='store_true', default=False,
         help='Similar to --start-with, starting with the package containing the current directory.')
+    add('--robust', action='store_true', default=False,
+        help='Try to build all packages instead of stopping if one failed.')
```

Comment:
What I want to avoid here is the confusion between:

- It will try to build all packages even if the packages they depend on failed to build and...
- It will try to build all packages which can be built (its dependencies built successfully), but failed packages and the packages which depend on them will not.

I'm not sure how we can achieve this will continuing to be brief. Maybe something like:

> Try to continue building packages whose dependencies built successfully even if some packages fail to build.